pages/articles/templates/articles/article_detail.html

```diff
@@ -0,0 +1,139 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+<!-- Main Content -->
+<div class="py-8">
+    <!-- Announcement (if present) -->
+    {% if article.announcement %}
+        <div class="bg-[#E5E5E5] rounded py-1.5 px-4 mt-0 mb-3 italic">
+            {{ article.rendered_announcement|safe }}
+        </div>
+    {% endif %}
+    
+    <!-- Title and Publication Date -->
+    <h1 class="text-3xl text-pp-almost-black mb-2">
+        {{ article.title }}
+    </h1>
+    <p class="text-pp-dark-grey mb-6">
+        Published: {{ article.created_at|date:"F j, Y" }}
+        {% if article.type == "editorial" and article.author %}
+            | Written by: {{ article.author }}
+        {% endif %}
+    </p>
+    
+    <!-- Featured Image (floated right on medium+ screens) -->
+    <figure class="float-none md:float-right mx-0 md:mx-2 mb-4 md:mb-0 w-96">
+        <img src="{{ article.featured_image.url }}" 
+             alt="{{ article.title }}"
+             class="w-full h-auto rounded border border-pp-pale-grey block">
+        {% if article.figure_caption %}
+            <figcaption class="text-sm text-pp-dark-grey mt-1 italic text-center block w-full break-words">
```

Comment:
The `tailwindcss` extension in VS code complains about `break-words` and asking to use `wrap-break-word`. They are essentially the same, I couldn’t find `break-words` in the tailwindcss docs, maybe it was an old utility meant to be replaced by [wrap-break-word](https://tailwindcss.com/docs/overflow-wrap#wrapping-mid-word) ?